charts/mimir-distributed/values.yaml

```diff
@@ -1235,38 +1235,38 @@ nginx:
           {{- end }}
 
           # Distributor endpoints
-          location /distributor {
+          location ~ /distributor {
             proxy_pass      http://{{ template "mimir.fullname" . }}-distributor.{{ .Release.Namespace }}.svc.cluster.local:{{ include "mimir.serverHttpListenPort" . }}$request_uri;
           }
           location = /api/v1/push {
             proxy_pass      http://{{ template "mimir.fullname" . }}-distributor.{{ .Release.Namespace }}.svc.cluster.local:{{ include "mimir.serverHttpListenPort" . }}$request_uri;
           }
 
           # Alertmanager endpoints
-          location {{ template "mimir.alertmanagerHttpPrefix" . }} {
+          location ~ {{ template "mimir.alertmanagerHttpPrefix" . }} {
             proxy_pass      http://{{ template "mimir.fullname" . }}-alertmanager.{{ .Release.Namespace }}.svc.cluster.local:{{ include "mimir.serverHttpListenPort" . }}$request_uri;
           }
-          location = /multitenant_alertmanager/status {
+          location ~ /multitenant_alertmanager {
```

Comment:
what other endpoints are there under multitenant_alertmanager?

Reply:
you can find them here https://github.com/grafana/mimir/blob/7f5bf4b7247af57279380226293946f535c43467/pkg/api/api.go#L189-L199

Reply:
I see some other, but our own document says not to expose to users: https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#alertmanager-configs